yaml/csr.yaml

```diff
@@ -17,26 +18,26 @@
 #
 ################################################################################
 #
-# CSR definitions for the CV32E40S CORE-V proessor core (an RV32IMCZ machine).
+# CSR definitions for the CV32E40S CORE-V processor.
```

Comment:
@Silabs-ArjanB, with the latest commit, I've addresses all your feedback except for these:

> Line 1391: E_EXT -> I_BASE_NONE

Why?

> Line 1703: msb: 3 (are there not checks for such overlapping declarations?)

Fixed it. But no, there seems to be no such checking.

> Line 2463: RW is misleading, same remark for mscratchcsw, mscratchcswl (I am not sure what is best, maybe WARL)

What should be done about it?

> Line 2871: Tied off bits missing

I didn't understand this one.

> Line 3265: This CSR is always there. 
> Line 3323 (and similar) All PMP CSRs are always present (only their values might differ)

Not sure what to do about this. Substitute all `COND PMP 1` etc with just `COND PMP`?

Reply:
> @Silabs-ArjanB, with the latest commit, I've addresses all your feedback except for these:
> 
> > Line 1391: E_EXT -> I_BASE_NONE
> 
> Why?

Because there is no such thing as an E extension. RV32E is a 'base' instruction set, not an extension.

> > Line 1703: msb: 3 (are there not checks for such overlapping declarations?)
> 
> Fixed it. But no, there seems to be no such checking.
> 
> > Line 2463: RW is misleading, same remark for mscratchcsw, mscratchcswl (I am not sure what is best, maybe WARL)

Just leave it like this as I don't think we have an accurate way to describe the behavior in the yaml format (maybe just add a comment about it).

> What should be done about it?
> 
> > Line 2871: Tied off bits missing
> 
> I didn't understand this one.

Should have been about line 2872, which is missing bits

> > Line 3265: This CSR is always there.
> > Line 3323 (and similar) All PMP CSRs are always present (only their values might differ)
> 
> Not sure what to do about this. Substitute all `COND PMP 1` etc with just `COND PMP`?

What is the number N in 'COND PMP N' supposed to mean? I don't think it corresponds to PMP_NUM_REGIONS as there are 4 regions per pmpcfg register. How do you want to use this COND PMP N comment?